src/File.hack

```diff
@@ -22,6 +22,45 @@ final class File {
     return $this->isDirty;
   }
 
+  private function getLines(): vec<string> {
+    return Str\split($this->contents, "\n");
+  }
+
+  const type TLineNumber1Base = int;
```

Comment:
Based not base

'base' by itself is a bit too confusing with number bases, e.g. base 16 == hex.

Reply:
`TLineNumber` feels sufficient though (comparing to 'line index')

Reply:
if you strongly feel that some extra clarity is needed `TLineNumber1First` is probably the close to the clearest